site/app/templates/grading/simple/Display.twig

```diff
@@ -23,60 +31,89 @@
-    {% endif %}
+        <div class="row-wrapper">
+            {#
+                Default is viewing your sections
+                Limited grader does not have "View All" option
+                If nothing to grade, Instructor will see all sections
+            #}
+            {% if show_all_sections_button %}
+                <a 
+                    class="btn btn-default"
+                    href="{{ core.buildUrl({'component': 'grading', 'page': 'simple', 'action': action, 'g_id': gradeable.getId(), 'sort': sort, 'view': view_all ? null : "all"}) }}"
+                >
+                    {{ view_all ? "View Your Sections" : "View All Sections" }}
+                </a>
+                <br />
+            {% endif %}
+
+            {% if action == 'lab' and core.getUser().accessFullGrading() %}
+                <label for="show-graders" class="column-wrapper">Show Graders
+                    <input type="checkbox" id="show-graders">
+                </label>
+                <label for="show-dates" class="column-wrapper">Show Dates Graded
+                    <input type="checkbox" id="show-dates">
+                </label>
+            {% endif %}
+        </div>
+    </div>
 
     {% if action == 'numeric' and core.getUser().accessAdmin() %}
-        <br> <br> <br>
-        <p style="float: right;">The CSV file should be formatted as such: <br />
-            user id,first name,last name,grade1,grade2,...,total points earned,text1,text2,...</p>
-        <input class="csvButtonUpload" type="file" id="csvUpload" style="float: right" accept=".csv, .txt">
-        <label for="csvUpload" style="float: right; padding-right: 10px;">Upload CSV</label>
+        <div class="column-wrapper">
+            <label for="csvUpload">Upload CSV
+                <input class="csvButtonUpload" type="file" id="csvUpload" accept=".csv, .txt"
+            </label>
+            <p>
+                The CSV file should be formatted as such: 
+                <br />
+                user id,first name,last name,grade1,grade2,...,total points earned,text1,text2,...
+            </p>
+        </div>
     {% endif %}
+
     {# This is a data table #}
-    <table class="table table-striped table-bordered persist-area" style="table-layout: fixed;">
+    <table id="data-table" class="table table-striped table-bordered persist-area" style="table-layout: fixed;">
         <caption />
         {# Table header #}
         <thead class="persist-thead">
             <tr>
                 <th width="20"></th>
                 <th width="70">Section</th>
                 <th width="90" style="text-align: left">
-                    <a href="{{ core.buildUrl({'component': 'grading', 'page': 'simple', 'action': action, 'g_id': gradeable.getId(), 'sort': 'id', 'view': view}) }}">
+                    <a href="{{ core.buildUrl({'component': 'grading', 'page': 'simple', 'action': action, 'g_id': gradeable.getId(), 'sort': 'id', 'view': view}) }}"
```

Comment:
are we supposed to be removing the closing `>` on this tag? (and the three other lines below)

Reply:
Good catch! I've pushed a fix.